docs/commandline.rst

```diff
@@ -4,3 +4,37 @@
 
 .. autoprogram:: doctr.__main__:get_parser()
    :prog: doctr
+
+
+Configuration
+-------------
+
+In addition to command line arguments you can configure ``doctr`` using the
+``.travis.yml`` files. Command line arguments take precedence over the value
+present in the configuration file.
+
+The configuration parameters available from the ``.travis.yml`` file often
+mirror their command line siblings except dashes ``-`` are replaced by
+underscores ``_``. Some parameters are not available from configuration files
+for security reasons.
+
+Use a ``doctr`` section in your ``travis.yml`` file to store your doctr
+configuration:
+
+.. code::
+
+  - language: python
+  - script: py.test
+  - doctr:
+    gh_pages_docs: 'documentation'
```

Comment:
This will change with https://github.com/drdoctr/doctr/pull/157 (which will be merged soon).